src/main/java/net/openhft/chronicle/network/ServerThreadingStrategy.java

```diff
@@ -17,10 +17,12 @@
  */
 package net.openhft.chronicle.network;
 
+@Deprecated(/* To be considered for removal in x.24 */)
```

Comment:
we normally give two releases warning, but if we are all OK with this then so am I!

Reply:
These comments are for us, they're not exposed to API users I believe and we have our own discretion whether remove or not.